### DISCUSSION TITLE
Clone and Publish -> Error

### DISCUSSION THREAD

**Discussion by andoniripollpro** [open-telemetry/opentelemetry-dotnet#4894] [Q&A]
Hi. 

I am having a hard time with all these OpenTelemery things. I consider this VS.Net solution like the Rosetta Stone to learn this. But I have a very basic error. If I just clone it and then I publish on VS.Net, it gives me an error:

Clone:
![opentelemetry-dotnet ps clone](https://github.com/open-telemetry/opentelemetry-dotnet/assets/9254576/d55b6ed4-b654-4e66-aa93-dcdec0400e96)

Publish: "We were unable to determine the cause of the error. Check the output log for more details.". No more details there
![image](https://github.com/open-telemetry/opentelemetry-dotnet/assets/9254576/ecbf3830-dfdc-49ae-8dbb-df9345a57ba9)

Can anyone help me with this?

Thanks in advance.

Regards,
Andoni

**Comment by pjanotti** [comment]
The intent is not for you to publish these projects, at most, I would expect that you may be able to publish some of the example or test applications. If you want to understand the implementation running the tests under the debug and stepping into the components will be more productive.

**Reply by andoniripollpro** [reply]
Thanks for the help, but what I need is to have some code that I´m sure it works and make it work with our collector, so I am sure I didn´t do anything wrong.
But any way, I copied the debug folder and I installed the framework on the machine I needed. The program worked but still the collector doesn´t get the logs

**Reply by pjanotti** [reply]
>  what I need is to have some code that I´m sure it works and make it work with our collector

I got the impression that you were trying to understand the implementation as a dev. To use OpenTelemetry .NET to send data to a collector you don't need to clone or build the repo. If you just want to set it up and receive the data with the collector check the instructions at https://opentelemetry.io/docs/instrumentation/net/

**Reply by andoniripollpro** [reply]
Thank you. The point is to undestand and develope my code from this. But I need first to understand a working app. I thank you for your help and I remake the question from another perspective.
